SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPossession/CrossServerPossessionGameMode.h

```diff
@@ -3,6 +3,7 @@
 #pragma once
 
 #include "CoreMinimal.h"
+#include "GameFramework/DefaultPawn.h"
```

Comment:
not necessary?

Reply:
Right, was left over from previous solution